packages/contact-center/store/src/store.ts

```diff
@@ -11,20 +19,31 @@ class Store {
     makeAutoObservable(this, {cc: observable.ref});
   }
 
-  init(webexConfig: any, access_token: string): Promise<void> { 
+  registerCC(webex: WithWebex['webex']): Promise<IAgentProfile> {
```

Comment:
I believe this will also change - only the type. Objects inside of the new `Profile` are pretty much the same, with a few additions.